docs/guides/validators/security_best_practices.md

```diff
@@ -473,6 +488,17 @@ We don't recommend limiting connections to this `port` (26656) because it is use
 laddr = "tcp://0.0.0.0:26656"
 ```
 
+Removing all other traffic 
+It is extremely important to use the DROP target for all other traffic **after** defining the --dport rules. This will prevent unauthorized access to the server through other open ports. To do this, simply type:
```

Comment:
Instead of explaining this in plain english, just provide a bash instruction set with correct order like it's done for ufw.